thunder/core/transform_common.py

```diff
@@ -111,6 +111,49 @@ def check(inp, log_str):
             check(copy_to_out, "output")
 
 
+def _replace_copy_output_with_source(extrace: Trace):
+    """The is based on the sharp edge of nvfuser's `add_ouput(output, input)` interface,
```

Comment:
I'm not quite sure but I guess this is typo of
```suggestion
    """This is based on the sharp edge of nvfuser's `add_ouput(output, input)` interface,
```